cmd/wallet-web/src/pages/chapi/Login.vue

```diff
@@ -19,10 +19,10 @@ SPDX-License-Identifier: Apache-2.0
             <md-card-content v-else>
                 <form>
                   <md-card-content>
-                    <md-button v-on:click="beginOIDCLogin" class="md-raised md-success" id="loginBtn">
-                      Login with Credentials
+                    <md-button v-on:click="beginOIDCLogin" class="md-raised md-success" id="loginBtn" style="text-transform: none">
```

Comment:
Just remove this change i have refactor this iin my pr to entirely different thing so kindly remove this file for this change.